sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -781,9 +784,13 @@ object GpuOverrides extends Logging {
   lazy val fileFormats: Map[FileFormatType, Map[FileFormatOp, FileFormatChecks]] = Map(
     (CsvFormatType, FileFormatChecks(
       cudfRead = TypeSig.commonCudfTypes + TypeSig.DECIMAL_128 +
-          GpuTypeShims.additionalCsvSupportedTypes,
+        GpuTypeShims.additionalCsvSupportedTypes,
       cudfWrite = TypeSig.none,
       sparkSig = TypeSig.cpuAtomics)),
+    (HiveDelimitedTextFormatType, FileFormatChecks(
+      cudfRead = TypeSig.commonCudfTypes + TypeSig.DECIMAL_128,
+      cudfWrite = TypeSig.commonCudfTypes + TypeSig.DECIMAL_128,
+      sparkSig = TypeSig.cpuAtomics)),
```

Comment:
On the read side the spark signature is for TypeSig.all.

https://github.com/NVIDIA/spark-rapids/blob/3909d7fc5007bd14cbe7e42f00f571cabe0523d7/sql-plugin/src/main/scala/org/apache/spark/sql/hive/rapids/HiveProviderImpl.scala#L136

I think we need to have the same here.

Reply:
Agreed. Modified.